scripts/import_libcbor.py

```diff
@@ -0,0 +1,64 @@
+import os
+import tempfile
+import shutil
+import subprocess
+import re
+
+# Create a temporary directory for cloning the repository
+temp_repo_dir = tempfile.mkdtemp()
+
+try:
+    # Clone the repository into the temporary directory
+    repo_url = "https://github.com/PJK/libcbor.git"
+    clone_command = f"git clone {repo_url} {temp_repo_dir}"
+    subprocess.run(clone_command, shell=True, check=True)
+
+    # Create a separate folder for the copied files
+    output_dir = os.path.join(
+        os.path.dirname(__file__), "..", "source", "external", "libcbor")
+    os.makedirs(output_dir, exist_ok=True)
+
+    # Copy files ending with .c and .h from the /src directory
+    src_dir = os.path.join(temp_repo_dir, "src")
+
+    # Use our customized configurations
+    src_customize_cbor_dir = os.path.join(
+        os.path.dirname(__file__), "libcbor_customize")
+    shutil.copytree(src_customize_cbor_dir, os.path.join(
+        output_dir, "cbor/"), dirs_exist_ok=True)
+
+    for root, dirs, files in os.walk(src_dir):
+        dir_name = os.path.basename(root)
+        for file in files:
+            if file.endswith((".c", ".h")):
+                src_file = os.path.join(root, file)
+                rel_path = os.path.relpath(src_file, src_dir)
+                dest_file = os.path.join(output_dir, rel_path)
+                os.makedirs(os.path.dirname(dest_file), exist_ok=True)
+                if dir_name == "cbor":
+                    with open(src_file, 'r') as f:
+                        contents = f.read()
+                    pattern = r'#include "cbor/'
```

Comment:
You can avoid doing any edits if you add the following to aws-c-common's CMakeLists.txt:
```
# libcbor files do includes assuming their at the root, like: #include "cbor/cbor_export.h"
# To make these paths work, add the location we're storing them as a search path.
target_include_directories(${PROJECT_NAME} PRIVATE
        ${CMAKE_CURRENT_SOURCE_DIR}/source/external/libcbor)
```